scss/tech.scss

```diff
@@ -0,0 +1,54 @@
+.tech {
+  padding: 10px 40px;
+}
+
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 150px;
+  left: 10px;
+  margin-bottom: 50px;
+  margin-right: 30px;
+  position: relative;
+  height: 280px;
```

Comment:
If there was a really long description setting a fixed height could break the layout. it's better to set a min-height to take into account those cases.